### PR TITLE
Add FILE_STRUCTURE.md.

### PR DESCRIPTION
**Pull request by tariksouabny**
Resolve Issue #37

### CODE REVIEW

**Comment by lahdjirayhan** [comment]
Awesome! Thanks for your contribution. I'm reviewing this shortly.

File: FILE_STRUCTURE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Classes
+In this repo, there are two main files where the classes are written. `ozone.py`, and `urls.py`. It is important to know the difference between them.
+
+### `urls.py`
+In this file, the classes stored provide the URL for the other files to use.
+
+### `ozone.py`
+In this file, the classes are stored with the purpose of creating data for the end user. The classes in this file depend on the classes in `urls.py`.
+
+This file generates the required token to access the data, makes a request to the API, retrieves the data, and parses it.
```
Comment (lahdjirayhan) [inline FILE_STRUCTURE.md:21]: No, Ozone does not **generate** any token. All users should obtain their own API keys (as explained in README.md) This sentence sounds like Ozone provides the API while in reality, Ozone doesn't.

File: FILE_STRUCTURE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Dependency Management
+### `setup.py`
+This file is used to set up the project and install the required dependencies.
+### `config.cfg`
```
Comment (lahdjirayhan) [inline FILE_STRUCTURE.md:26]: Do you mean `setup.cfg` as opposed to `config.cfg`?

**Review by lahdjirayhan — CHANGES_REQUESTED** [review]

**Comment by tariksouabny** [comment]
Thanks for the review! I made the suggested changes.

**Review by Milind220 — CHANGES_REQUESTED** [review]
Your pull request is currently trying to merge in from your fork's main branch to Ozone's main branch. The contributing guidelines mention that PR's are supposed to be made to the dev branch of Ozone. Please edit your PR's target branch to dev

**Comment by lahdjirayhan** [comment]
@tariksouabny Looking better to me!

**Review by Milind220 — APPROVED** [review]
That's great, I'll merge it in now!

**Pull request merged**